man/systemd.socket.xml

```diff
@@ -216,6 +216,11 @@
         <varname>BindIPv6Only=</varname> setting (see below).
         </para>
 
+        <para>If the address string is a string in the format
+        vsock:x:y, it is read as CID x on a port y address in the
```

Comment:
the `vsock:x:y` should be enclosed in `<literal></literal>` I figure...

Hmm, what's a CID precisely? I think it would be nice to briefly explain that in brackets or so.

`x` and `y` should be enclosed in `<literal></literal>` too I figure.